doc/examples/denoise_nlmeans.py

```diff
@@ -10,36 +10,28 @@
 """
 
 import numpy as np
-import nibabel as nib
 import matplotlib.pyplot as plt
 from time import time
 from dipy.denoise.nlmeans import nlmeans
 from dipy.denoise.noise_estimate import estimate_sigma
 from dipy.data import get_fnames
-from dipy.io.image import load_nifti
+from dipy.io.image import load_nifti, save_nifti
 
 
-dwi_fname, dwi_bval_fname, dwi_bvec_fname = get_fnames('sherbrooke_3shell')
-data, affine = load_nifti(dwi_fname)
+t1_fname = get_fnames('stanford_t1')
+data, affine = load_nifti(t1_fname)
 
-mask = data[..., 0] > 80
-
-# We select only one volume for the example to run quickly.
-data = data[..., 1]
+mask = data > 1500
 
 print("vol size", data.shape)
 
-# lets create a noisy data with Gaussian data
-
 """
 In order to call ``non_local_means`` first you need to estimate the standard
-deviation of the noise. We use N=4 since the Sherbrooke dataset was acquired
-on a 1.5T Siemens scanner with a 4 array head coil.
+deviation of the noise. We use N=32 since the Standford dataset was acquired
```

Comment:
_Standford_ should be _Stanford_

Reply:
Good catch! 

@skoudoro : the CI failures seem unrelated to this. Do you understand what those are all about? Looks like something to do with `deform_streamlines`. Have you seen this one before?

Reply:
thanks @jhlegarreta!

Yes, I have seen it before @arokem, I don't the origin yet but usually, I just restart the matrix and it passes.
For Now, I just rebase to see if the Codecov comment will appear or not.

Let's wait for the CI to finish and you can go ahead. Thanks!

Reply:
It needed 2 restarts -- everything seems green now :)

Reply:
Thanks @skoudoro and all for having done this.